chapters/exploitation-techniques/shellcodes/drills/06-challenge-binsh-shellcode/src/Makefile

```diff
@@ -1,21 +1,22 @@
-CFLAGS = -Wall -fno-stack-protector -g -fno-PIC
-LDFLAGS = -no-pie
+PORT ?= 31344
+IMG_NAME ?= challenge06
+CONT_NAME ?= $(IMG_NAME)-cnt
 
-.PHONY: all clean
+build:
+	docker build -t $(IMG_NAME) -f Dockerfile .
+	make -f Makefile.sol
```

Comment:
Remove this step. The use of the `Makefile.sol` is to be done inside the `Dockerfile`. See my comment in the `Dockerfile`.